invenio_rdm_records/fixtures/__init__.py

```diff
@@ -10,6 +10,10 @@
 
 from pathlib import Path
 
+import yaml
+
+from .affiliations import AffiliationsFixture
+from .fixture import FixtureMixin
```

Comment:
cannot put it in the `__init__.py` because it generates a circular dep 😢

Reply:
Not really, that's what's causing the circular dependency:
- `__init__.py` imports `from .affiliations import AffiliationsFixture`
- `affiliations.py` imports `from .fixture import FixtureMixin`

If we move the engine to `fixture.py` that would mean moving the first import there and then we would have `fixture.py` -> `affiliations.py` -> `fixture.py` and so on

Reply:
Ah yes, a `fixture_engine.py` file is missing here. 

```
fixtures/
+-- __init__.py (pretty much empty except for __all__ and relevant imports)
+-- fixture.py with the FixtureMixin
+-- affiliations.py with AffiliationsFixture that imports FixtureMixin from fixture.py
+-- fixture_engine.py with the FixtureEngine that imports AffiliationsFixture
```

but that's mostly to turn that :cry: into :smiley: and to make `__init__.py` not carry any functionality...